projects/stargazer/plugins/other/radius/server.cpp

```diff
@@ -48,10 +53,21 @@ RadProto::Packet Server::makeResponse(const RadProto::Packet& request)
     std::vector<uint8_t> vendorValue {0, 0, 0, 3};
     vendorSpecific.push_back(RadProto::VendorSpecific(m_dictionaries.vendorCode("Dlink"), m_dictionaries.vendorAttributeCode("Dlink", "Dlink-User-Level"), vendorValue));
 
-    if (request.type() == RadProto::ACCESS_REQUEST)
-        return RadProto::Packet(RadProto::ACCESS_ACCEPT, request.id(), request.auth(), attributes, vendorSpecific);
+    std::vector<RadProto::Attribute*> attributesEmpty;
+    std::vector<RadProto::VendorSpecific> vendorSpecificEmpty;
 
-    return RadProto::Packet(RadProto::ACCESS_REJECT, request.id(), request.auth(), attributes, vendorSpecific);
+    if (request.type() == RadProto::ACCESS_REQUEST)
```

Comment:
In cases like this:
```c++
if (condition)
{
    // A lot of code
    return something;
}
return someOtherThing;
```
consider turning inside out, like this:
```c++
if (!condition)
    return someOtherThing;
    
// A lot of code
return something;
```

It looks cleaner, it is easier to understand and it can reduce cyclomatic complexity.